Refined monorail system

UzSid opened the pull request:
### New Track Model
<img width="1195" height="701" alt="image" src="https://github.com/user-attachments/assets/50fe2b4c-e891-4ba5-a74f-2e0f9082cbe2" />
<img width="1291" height="820" alt="image" src="https://github.com/user-attachments/assets/7dfa5c3c-336a-4ba7-b257-71560cc8dc01" />

Added support pillars for the track:
<img width="788" height="1154" alt="image" src="https://github.com/user-attachments/assets/d40d659d-3424-4e00-a4ef-d280da20f917" />


### New Train Model
<img width="1010" height="891" alt="image" src="https://github.com/user-attachments/assets/60fe6f38-8822-46e5-9c03-9cca8f30b4bf" />

### Improved Train Mechanics
- The train will explode upon colliding with an object after derailing
   - If this does not work, it will explode after a delay
- Increased default train speed to 70%
- The train will stop after passing by these markers (if start/stop is enabled) instead of relying on timing:
<img width="740" height="476" alt="image" src="https://github.com/user-attachments/assets/1836a9be-0c20-466a-a37f-144976dd4b75" />

   - If the train is moving at the default speed, it will stop around the center of the station
      - If the train is moving too fast, it will stop too late

Nenshki reviewed (changes requested):
The new assets look great! The stoppers appear to work properly & the derailment is still fun to watch.

There were a few issues I found stated below:

_Major issues_
- When stopped, the monorail still continues to play the sound like it's still moving (If possible, maybe have this slowly fade out into silence, or maybe include a slow down SFX)
- The volume of the screams & when the monorail is traversing the track is a bit loud for the distance the player can hear them (It only gets worse the closer I get to the monorail)

_Minor Issues (These can be solved later)_
- This is a suggestion, but I think the frequency of the screams are happening too quick in succession. Since the screams start at a certain speed not at a critical point of derailment, I can imagine players getting annoyed with the constant screaming (Maybe spacing out the individual screams slightly would help with noise reduction) (Though this may be dependent on how we implement the hacking from the back-end)
- The time for the monorail staying at a station may need to be extended (Game-play wise, the player needs to have enough time to perform the intended hack) (Since this part of the system hasn't been figured out yet, ignore for now, but keep it in mind)

UzSid commented:
- I fixed the issue of the train continuing to play its sound while stopped. Unfortunately, I was unable to make it so that it fades out while also making the sound increase or decrease based on the speed of the train (i.e. it's quieter when the speed is low and louder when the speed is fast). I think finding a good slowdown sound is the easiest way to make the transition smoother.
- I reduced the volume of the train and screams.
- I increased the delay between screams.
- I added more scream sounds to make it feel less repetitive.
- I adjusted the track so that the gas station sign is not obstructed:
<img width="1207" height="892" alt="image" src="https://github.com/user-attachments/assets/f995e8f8-0c33-476d-b3a1-3b8dbe3bef6a" />

Nenshki reviewed (approved):
All stated fixes proved to be satisfactory. The monorail volume should be at a good enough level for most devices & the new delay of the screams is at a good pace (I found that it may even have comically perfect timing with the explosion). All good for merging!
